### ISSUE TITLE
Cannot do `:diffthis`

### ISSUE THREAD

**Issue by wookayin** [dstein64/nvim-scrollview#57]
When this plugin is enabled, I cannot do `:diffthis`. Without the plugin, everything works like normal.

Reproduce:

```
vim -O file1 file2
:diffthis                 " on file 1
" Move to another window
```

Then the `&diff` option for the window is lost. `WinLeave` triggers `:lua require('scrollview').refresh_bars_async()`, and during the execution of this lua function `&diff` will be lost.

**Comment by dstein64** [comment]
Thanks for reporting the issue @wookayin!

This is fixed in fb35d9d7b52985baac5d77ec52d6b6b35403fb8a.

**Comment by wookayin** [comment]
Works well as expected after the update. Thank you!
